src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" method="get" action="index.html">
```

Comment:
```suggestion
    <form class="form" method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
```

Reply:
I have to add a title?
I added earlier, I was told that it is not in the example. So I deleted it